### PR TITLE
Fix typo

### PR DESCRIPTION
**Pull request by Gouthamve**
This trips up everyone I show the repo to :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,4 +2,4 @@
 
 [![Build Status](https://travis-ci.org/improbable-eng/thanos.svg?branch=master)](https://travis-ci.org/improbable-eng/thanos) [![Go Report Card](https://goreportcard.com/badge/github.com/improbable-eng/thanos)](https://goreportcard.com/report/github.com/improbable-eng/thanos)
 
-A block storage based long term storage for Prometheus.
\ No newline at end of file
+A object storage based long term storage for Prometheus.
```
Comment (Bplotka) [inline README.md]: `An object storage`? (:

Comment (Gouthamve) [reply README.md]: Done :)

**Comment by fabxc** [comment]
Yes :D I constantly swap those words for some reason. Already getting fixed in #122 though

**Comment by Gouthamve** [comment]
Perfect!

**Pull request closed without merging**